pkg/resources/configmaps/configmap.go

```diff
@@ -0,0 +1,58 @@
+package configmaps
+
+import (
+
+	hawtiov1alpha1 "github.com/hawtio/hawtio-operator/pkg/apis/hawtio/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"bytes"
+
+	"text/template"
+)
+
+// Create NewConfigMapForCR method to create configmap
+func NewConfigMapForCR(m *hawtiov1alpha1.Hawtio) *corev1.ConfigMap {
+	config := ConfigForHawtio(m)
+	configMap := &corev1.ConfigMap{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: "apps/v1",
+			Kind:       "ConfigMap",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      m.Name,
+			Namespace: m.Namespace,
+		},
+		Data: map[string]string{
+			"hawtconfig.json": config,
+		},
+	}
+
+	return configMap
+}
+
+func ConfigForHawtio(m *hawtiov1alpha1.Hawtio) string {
+	config := `
+ {
+        "about": {
+          "title": "Hawtio Console",
+          "productInfo": [],
+          "additionalInfo": "The Hawtio console eases the discovery and management of 'hawtio-enabled' applications deployed on OpenShift.",
+          "copyright": ""
+        },
+        "branding": {
+          "appName": "Hawtio Console",
+          "appLogoUrl": "img/hawtio-logo.svg",
+          "consoleLink": {
+            "text": "Hawtio Console",
+            "section": "Hawtio",
+            "imageRelativePath": "/online/img/favicon.ico"
+          }
+        },
+        "disabledRoutes": []
+      }`
+	var buff bytes.Buffer
+	hawtioconfig := template.Must(template.New("hawtioconfig").Parse(config))
+	hawtioconfig.Execute(&buff, m.Spec)
+	return buff.String()
+}
```

Comment:
There are a couple of values that are changed here as part of the product build, like the `title`, `additionalInfo`, `copyright`, `appName`, `appLogoUrl` and `disabledRoutes` fields. It used to be performed by _seding_ the template file. What could be the best option now with code-generated config? I can think of Go build variables that we already use for the Fuse Console image build, but it may be cumbersome to have one variable per field. WDYT?

Reply:
@astefanutti then can we put this config string into YAML file and load from there?

Reply:
value-added in YAML file and will be retrieved from [config folder](https://github.com/redhatHameed/hawtio-operator/tree/remove-template/config)

Reply:
@redhatHameed this is great. Thanks!